submit/helpers.py

```diff
@@ -274,21 +273,16 @@ def fetch_request_ids(item_ids, cls, attr_name, verification_list=None):
     return items
 
 
-def fetch_name_by_umail(umail, request):
+def fetch_name_by_umail(umail):
     def extract(item):
         if len(data[item]) == 1:
             return data[item][0]
         raise Exception('Multiple values returned: {}'.format(data))
 
     uid = umail.split('@')[0]
 
-    # Return the portion before @umail.ucsb.edu if ldap_uri is not provided
-    ldap_uri = request.registry.settings.get('ldap_uri')
-    if not ldap_uri:
-        return uid
-
     # connect to ldap
-    ldap_conn = ldap.initialize(ldap_uri)
```

Comment:
Looks like this is code added by Bryce to assist with running, for example, on AWS, outside of the context where the UCSB ldap server is directly accessible.